apps/web/app/actions/general.ts

```
@@ -84,11 +84,19 @@ export async function subscribeToNewsletter(
 }
 
 // generate dub.co links
-export async function generateShortLink(url: string) {
-  const cookieStorage = cookies()
+export async function generateShortLink(url: string, withCookies = true) {
+  // const cookieStorage = cookies()
+  let cookieStorage;
+  let getShareLinkCookies;
```

Biome

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)